catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -397,8 +399,19 @@ private void setProxiesForImagery(List<Map<String, Object>> newImageryProviders)
         proxiedImageryProviders.clear();
         for (Map<String, Object> newImageryProvider : newImageryProviders) {
             HashMap<String, Object> map = new HashMap<>(newImageryProvider);
-            map.put(URL, SERVLET_PATH + "/" + urlToProxyMap.get(newImageryProvider.get(URL)
-                    .toString()));
+            String baseUrl = newImageryProvider.get(URL).toString();
+            boolean proxyEnabled = true;
+            Object proxyEnabledProp = newImageryProvider.get(PROXY_ENABLED);
+            if (proxyEnabledProp instanceof String) {
+                proxyEnabled = Boolean.parseBoolean((String)proxyEnabledProp);
+            }
+
+            if (proxyEnabled) {
+                map.put(URL,
+                        SERVLET_PATH + "/" + urlToProxyMap.get(baseUrl));
+            } else {
+                map.put(URL, baseUrl);
+            }
             proxiedImageryProviders.add(map);
```

Comment:
Should we change the variable names to be more representative so that someone out of context can follow?  proxiedImageryProviders are really just imageryProviders now right, may or may not be proxied.  

And is the baseUrl really a baseUrl?  Or is it more the originalUrl or just url or imageryProviderUrl or providerUrl?